### DISCUSSION TITLE
Radio component

### DISCUSSION THREAD

**Discussion by ivanov-v** [ariakit/ariakit#2214] [Questions] [answered]
Hi. Why I should use `Radio` component instead of `<input type="radio">`?
Unfortunately, I did not find a detailed description in the documentation. Are you planning to add this?

**Comment by diegohaz** [comment, marked as answer]
It gives you all the behaviors and attributes described in the [Radio Group Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/radiobutton/) for both native and non-native radio buttons, including support for [roving tabindex](https://www.w3.org/WAI/ARIA/apg/example-index/radio/radio.html) and [aria-activedescendant](https://www.w3.org/WAI/ARIA/apg/example-index/radio/radio-activedescendant.html).

**Reply by ivanov-v** [reply]
Will you add a detailed description to the documentation for uninformed users?)
